docs/topics_5/web-openshift-deploy.adoc

```diff
@@ -9,18 +9,16 @@ The path to the directory created by unzipping this file is referred to as `_MTA
 +
 image::openshift-console.png[]
 
-. Click the `Import YAML / JSON` button in the top-right corner of the web console.
-. Provide the following information in the window that appears, as seen below.
-+
-image::openshift-import-template.png[]
-.. Define a `Project Name`, such as `mta`.
-.. Optionally, define a `Project Display Name` to help describe the project.
-.. Optionally, define a `Project Description` to help provide context on how the project will be used.
-.. Click the **Browse** button and import the desired {WebName} template. These are included in `__MTA_HOME__/openshift/templates`. Once imported, the JSON is visible in the bottom text area.
-. Click the **Create** button to proceed to the next screen.
+. Click *Import YAML / JSON* in the top-right corner of the web console.
+. Specify:
```

Comment:
Use complete sentences. Don't use parenthetical phrases or statements.

Fill in the following fields:
* *Project Name*: Mandatory
* *Project Display Name*: Optional. This field determines how the project name is displayed in the web console.
* *Project Description*: Optional
* *Template*: Mandatory. Click *Browse* and navigate to ....

I'm guessing about the name of the Template field. You should check with Phil because it might not be visible in the old screenshot.
*